docs/docs/01-ibc/01-overview.md

```diff
@@ -34,21 +34,22 @@ The requirements to have your module interact over IBC are:
 - Use the default acknowledgment struct provided by core IBC or optionally define a custom acknowledgment struct.
 - Standardize an encoding of the packet data.
 - Implement the `IBCModule` interface.
+- Implement the `UpgradableModule` interface.
```

Comment:
Right now this is optional, we discussed making it required for v9 but I kind of like having it optional too

Reply:
Made it optional.